src/Context/SoapContext.php

```diff
@@ -104,8 +74,7 @@ public function iAmWorkingWithSoapServiceWithOptionsYaml(PyStringNode $options)
      */
     public function iSendRequestWithParams($function, TableNode $params)
     {
-        $this->arguments = array($params->getRowsHash());
-        $this->sendRequest($function, $this->arguments);
+        $this->sendRequest($function, [$params->getRowsHash()]);
```

Comment:
I bet someone can prepare $this->arguments in advance and then use those to do the soap request (or reuse arguments and append something to it maybe on a second call, that is why it was a instance property, but maybe we refactor this later when we add that support